test/test-remote/test_loki_log_ingest_api.ts

```diff
@@ -198,6 +201,53 @@ suite("Loki API test suite", function () {
     );
   });
 
+  test("fluentbit(grafana loki plugin) with basic auth", async function () {
+    const loglines = ["aaaaaaa", "bbbbbb", "cccccccc"];
+
+    const idxfieldname = "indexfieldname";
+    const idxfieldvalue = rndstring().slice(0, 5);
+
+    await sendLogsWithFluentbitContainer(
+      `loki.default.${OPSTRACE_INSTANCE_DNS_NAME}`,
+      TENANT_DEFAULT_API_TOKEN_FILEPATH,
+      loglines,
+      idxfieldname,
+      idxfieldvalue,
+      "fbit.config.template",
+      // rely on the fbit stdout output plugin to be used, i.e. all log lines
+      // appear on the container's stdout once read.
```

Comment:
Note: when the needle appears in the log output this just means that the `tail` input plugin has processed the data, but it does not mean that the output plugin has flushed the data out to loki. That is, when the subsequent query fails it's important to inspect the container log for error detail.

I have seen such error detail to be logged for 4xx responses _before_ the container is actually terminated. For timeout issues etc, it might happen that we terminate the container before error detail is logged. Just a heads-up for debugging when this breaks.